### DISCUSSION TITLE
Base French Translations

### DISCUSSION THREAD

**Discussion by volcanofr** [facebook/docusaurus#8343] [General]
Hello @forresst !

I have a small question : why we are using the English "tags" and not the french "étiquettes" [here](https://github.com/facebook/docusaurus/blob/main/packages/docusaurus-theme-translations/locales/fr/theme-common.json) ?

**Comment by forresst** [comment]
Good question. Many websites use "tags" in French. I'm not against changing to something else. @slorber your opinion ?

**Reply by slorber** [reply]
I also prefer using anglicisms for such things, particularly when it doesn't translate so well and the french word is so much longer.

This will remain subjective and whatever we decide it won't please everybody. The good news is that you can override it so it shouldn't be a problem to keep it this way.
